workshop/SUMMARY.md

```diff
@@ -12,7 +12,6 @@
 * [Machine Learning with AutoAI](machine-learning-autoai/README.md)
 * [Deploy and Test Machine Learning Models](machine-learning-deployment-scoring/README.md)
 * [Monitoring models with OpenScale GUI (Fastpath Monitoring)](openscale-fastpath/README.md)
-* [(Optional) Monitoring models with OpenScale (Notebook)](openscale-notebook/README.md)
```

Comment:
@scottdangelo We want to showcase the manual config (openscale-manual-config) this time around, no ?

Reply:
I'll add the manual config. Should I delete the fastpath?

Reply:
I would keep both.